packages/libs/error-stack/tests/test_compatibility.rs

```diff
@@ -51,18 +49,21 @@ fn anyhow() {
 
     #[allow(unused_mut)]
     let mut report_messages = messages(&report);
+
+    // Backtrace from anyhow cannot be captured currently
```

Comment:
Do we perhaps want to quantify "currently"? Are we expecting this to change in a future version?

Reply:
Added a description in 690dc8873769af9407f971029862d5a485b867e0.
In the case `anyhow` will add support, this won't affect our tests, users of error-stack will implicitly get the backtrace